web/src/containers/loans/Loans.js

```diff
@@ -476,6 +476,10 @@ const FundingLoans = (props) => {
                     tuition={Number(
                       BigInt(l.requested_value_atto_dai) / BigInt(10 ** 18)
                     )}
+                    fundedPercentage={Number(
+                      BigInt(l.funded_value_atto_dai * 100) /
+                        BigInt(l.requested_value_atto_dai)
+                    )}
```

Comment:
move convertions to loans.service

Reply:
Why tho? That way we would have to go over the array 2 times, instead of going only one